### ISSUE TITLE
problems about Gaussian classification

### ISSUE THREAD

**Issue by gaoxiaoyao** [cornellius-gp/gpytorch#1187]
Hello, I am a novice in learning Gaussian process. I use the method of Gaussian classification to do the research of MNIST digital recognition. Now after training the model, I want to output the recognition accuracy of the model, but I have encountered difficulties in output. I am prompted with the tensor problem. Can you help me to see how to modify the output accuracy? Thank you for your advice. For convenience, I put the compressor below:
num_epochs=3
for i in range(num_epochs):
scheduler.step()
for minibatch_i, (x_batch, y_batch) in enumerate(train_loader):
    optimizer.zero_grad()
    output = model(x_batch)
    log_lik, kl_div, log_prior = mll(output, y_batch)
    loss = -(log_lik - kl_div + log_prior)
    print('Epoch %d [%d/%d] - Loss: %.3f [%.3f, %.3f, %.3f]' % (i + 1, minibatch_i, len(train_loader), loss.item(), log_lik.item(), kl_div.item(), log_prior.item()))
    loss.backward()
    optimizer.step()
[multiclass_classifier_for_issue.zip](https://github.com/cornellius-gp/gpytorch/files/4801967/multiclass_classifier_for_issue.zip)

**Comment by gpleiss** [comment]
What kind of difficulties are you encountering? Are you getting some sort of error?

**Comment by gaoxiaoyao** [comment]
# Evaluate accuracy
model.eval()
likelihood.eval()
n_test = 10000
n_gpu_can_take = 200
test_pred_labels = torch.Tensor([])
for i in range(0,n_test,n_gpu_can_take):
    with torch.no_grad(), gpytorch.settings.fast_pred_var():
        predictions = likelihood(model(test_x[i:i+n_gpu_can_take].cuda()))
        test_pred_labels = torch.cat((test_pred_labels,predictions.probs.max(dim=1)[1].float().cpu()))
(test_pred_labels == test_y[:n_test]).float().mean()
There is no problem with this model training, but the model accuracy cannot be displayed. So I would like to ask how to display the model accuracy，thank you

**Comment by gpleiss** [comment]
`(test_pred_labels == test_y[:n_test]).float().mean()` should be the model accuracy.
